frontend/src/screens/learner.tsx

```diff
@@ -281,6 +322,7 @@ export const StudiesContainer = () => {
                         </Group>
 
                         <SearchResults search={search} filteredStudies={filteredStudies} />
+                        <Divider my="-5"  />
```

Comment:
There is a lot of gap between the Search Container and the and the Divider. As per the design spec it should be smaller.